docs/application/vstools/tools/emulator-control-panel.md

```diff
@@ -0,0 +1,487 @@
+# Emulator Control Panel
+
+Before deploying your application, it is important that you test it in an environment similar to a real device.
+
+You can run the application in the emulator, and test a variety of user scenarios, such as network access, audio input and out, and text messages. With a mouse and keyboard, you can control the application in the emulator just like on an actual device.
+
+While the application is running, you can use the Emulator Control Panel to simulate events for a variety of system options that the actual device provides. For example, by manipulating the virtual battery, you can simulate the application in different charge environments.
+
+The emulator controls consist of internal and external parts:
+
+-   The **HOME**, **Volume control** and **Back** buttons, for example, are external parts controlling the device from the outside.
+
+    In the emulator, the external parts are called the **emulator control keys and menu**.
+
+-   Battery level and screen brightness, for example, are internal parts controlling the device from the inside. In the emulator, the internal parts are called the **Emulator Control Panel**.
+
+
+## Using the Control Keys and Context Menu
+
+The control keys are visible on the emulator when you start it. To access the context menu, right-click the emulator.
+
+**Figure: TV, mobile, and wearable emulators**
+
+**![TV, mobile, and wearable emulators](media/emulator_tv_mobile.png)**
+
+> **Note**  
+> You can create a custom resolution emulator by using the [Emulator Manager CLI](../../tizen-studio/common-tools/emulator-manager.md#control), and launch it with the TV and mobile skin. It is not guaranteed that all applications are correctly shown in the custom resolution.
```

Comment:
It looks like you updated the names used in `<a name>` in the emulator-manager.md, so `emulator-manager.md#control` is leading to wrong point. It looks like it should be changed into `emulator-manager.md#user-content-control`. Would you check?

Reply:
The other member is updating tizen-studio docs. If the task is finished, the hyperlink will be fixed automatically.